src/AIconLabel.cpp

```diff
@@ -9,10 +9,23 @@ AIconLabel::AIconLabel(const Json::Value &config, const std::string &name, const
                        bool enable_click, bool enable_scroll)
     : ALabel(config, name, id, format, interval, ellipsize, enable_click, enable_scroll) {
   event_box_.remove();
+  label_.unset_name();
+  label_.get_style_context()->remove_class(MODULE_CLASS);
+  box_.get_style_context()->add_class(MODULE_CLASS);
+  if (!id.empty()) {
+    label_.get_style_context()->remove_class(id);
+    box_.get_style_context()->add_class(id);
+  }
+
   box_.set_orientation(Gtk::Orientation::ORIENTATION_HORIZONTAL);
-  box_.set_spacing(8);
+  box_.set_name(name);
+
+  int spacing = config_["icon-spacing"].isInt() ? config_["icon-spacing"].asInt() : 6;
```

Comment:
Mhmm 8 was the default before the change, should be set to 8

Reply:
My bad, I've amended this in my most recent push.